tests/coordination_test.py

```diff
@@ -0,0 +1,8 @@
+
+def test_coordination_T(example_gene_coor,res_coor):
+    res=example_gene_coor.gene_coordination_test(test="fisher")
+    assert all([res[i]==res_coor[i] for i in range(2,13)])
```

Comment:
maybe also test p-value >0 and pvalue <.05

Reply:
Added this as well.